### PR TITLE
Add username to todo list

### PR DESCRIPTION
**Pull request by FionaDL**
This PR lets the user know who they are signed in as. 

https://github.com/fastruby/dash/issues/4
Screen Shot 2020-05-28 at 4 04 09 PM

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -99,7 +99,15 @@ form{
 }
 
 #todo_title {
-  color: white;
+  color: #F70A7C;
```
Comment (arielj) [inline app/assets/stylesheets/application.scss:102]: You should use sass variables here so you don't hardcode the color provided by the styleguide

```suggestion
  color: $magenta;
```

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -99,7 +99,15 @@ form{
...
+  font-weight: 700;
   float: left;
-  font-size: 28px
+  font-size: 28px;
+  font-family: "open sans";
```
Comment (arielj) [inline app/assets/stylesheets/application.scss:106]: Same here:

```suggestion
  font-family: $font-open-sans;
```

(this files defines the variables you can use https://github.com/ombulabs/styleguide/blob/gh-pages/vendor/assets/stylesheets/ombulabs/styleguide/1-utilities/_variables.scss)

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -99,7 +99,15 @@ form{
...
 }
+
+// #username {
```
Comment (arielj) [inline app/assets/stylesheets/application.scss:109]: Don't leave this commented, if it's not needed just remove it

**Comment by etagwerker** [comment]
@arielj I can't see these changes in staging, did you deploy them to that environment? 

Screen Shot 2020-05-30 at 10 31 43 AM

**Comment by arielj** [comment]
> @arielj I can't see these changes in staging, did you deploy them to that environment?
> 
> Screen Shot 2020-05-30 at 10 31 43 AM

No, sorry, I didn't, I was confused with the last steps about who had to deploy to staging. I still feel there's something off there on our current PR review process though, I have to deploy something to staging not knowing if there's something specific on staging at the moment. Right know there's only one PR but if fiona had 2 PRs pending at the same time which one should be deployed to staging?

Can we try heroku's review apps at least on one project to see if this last part of the process is more clear? Having heroku deploy PRs automatically to their own isolated app on would prevent this things to happen and we won't need to manually do something that can be done automatically and free by heroku

**Comment by etagwerker** [comment]
> Right know there's only one PR but if fiona had 2 PRs pending at the same time which one should be deployed to staging?

Please talk to Fiona and take turns. 

> Can we try heroku's review apps at least on one project to see if this last part of the process is more clear?

No, for this project it’s overkill. You and Fiona can talk and take turns on staging. It’s not that hard. 

Review apps in Heroku are great but only when there are too many developers making too many changes that need to be tested. This application doesn’t need it right now.

**Comment by arielj** [comment]
> > Right know there's only one PR but if fiona had 2 PRs pending at the same time which one should be deployed to staging?
> 
> Please talk to Fiona and take turns.
> 
> > Can we try heroku's review apps at least on one project to see if this last part of the process is more clear?
> 
> No, for this project it’s overkill. You and Fiona can talk and take turns on staging. It’s not that hard.
> 
> Review apps in Heroku are great but only when there are too many developers making too many changes that need to be tested. This application doesn’t need it right now.

Maybe we should update the proposed review process stating that only one PR can be labeled Ready to QA at a time? cc @ABizzinotto 

Just deployed this to staging

**Comment by etagwerker** [comment]
> Maybe we should update the proposed review process stating that only one PR can be labeled Ready to QA at a time?  

@arielj This is a great idea.

@FionaDL is the color change approved by @nykka? I'm not sure it looks better than before... and I don't see it as a change request over here: https://github.com/fastruby/dash/issues/4 

## Before 

Screen Shot 2020-06-01 at 10 27 44 AM

## After 

Screen Shot 2020-06-01 at 10 28 22 AM

**Comment by FionaDL** [comment]
> > Maybe we should update the proposed review process stating that only one PR can be labeled Ready to QA at a time?
> 
> @arielj This is a great idea.
> 
> @FionaDL is the color change approved by @nykka? I'm not sure it looks better than before... and I don't see it as a change request over here: #4
> 
> ## Before
> Screen Shot 2020-06-01 at 10 27 44 AM
> 
> ## After
> Screen Shot 2020-06-01 at 10 28 22 AM

@etagwerker it is approved by @nykka, but I'm not sure about it still either...You think it looks better in white?
Screen Shot 2020-06-01 at 10 59 58 AM

**Comment by etagwerker** [comment]
@FionaDL I think it looks a little better in white, could you please switch it back?

**Comment by FionaDL** [comment]
> @FionaDL I think it looks a little better in white, could you please switch it back?

@etagwerker yup, done.

**Comment by etagwerker** [comment]
@FionaDL No need. Thanks!

**Pull request merged**